src/components/ComparisonModal/comparisonModal.css

```diff
@@ -0,0 +1,160 @@
+.modal-container{
+  position: fixed;
+  width: 100vw;
+  height: 100%;
+  margin: 0;
+  background: var(--shadow-bg);
+  cursor: pointer;
+  z-index: 10;
+}
+
+
+.card{
+  width: 90%;
+  margin: 5rem auto;
+  background: var(--gray-bg);
+  border-radius: 1rem;
+  cursor: auto;
+}
+
+.pokemon-card__title{
+  display: flex;
+  height: 3.5rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;  
+}
+
+.pokemon-card__compare{
+  cursor: pointer;
+  border-radius: 0.3rem;
+  padding: 0.2rem 1rem;
+  margin-left: 1.5rem;
+  border: solid 1px var(--white);
+  background: var(--redish);
+  color: var(--white);
+}
+
+.pokemon-card__compare:hover,
+.pokemon-card__compare:active{
+  background: var(--white);
+  color: var(--redish);
+}
+
+.pokemon-card__close{
+  margin-left: auto; 
+  margin-right: 1.5rem;
+  border-radius: 50%;
+  cursor: pointer;
+  color: var(--redish);
+  border: solid 1px var(--redish);
+}
+
+
+
+
+.pokemon-container{
+  margin: 2rem 1.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: var(--white);
+  border-radius: 1rem;
+}
+
+.comparison-image-container{
+  width: 80%;
+  display: flex;
+  justify-content: center;
+  image-rendering: pixelated;
+  margin-top: -6rem;
+}
+
+.comparison-pokemon-img{
+  width: 10rem;
+}
+
+.pokemon-name{
+  margin: 0;
+  text-align: center;
+  color: var(--gray);
+}
+
+
+.information-container{
+  display: flex;
+  width: 80%;
+  text-align: center;
+  margin-top: 1rem;
+  justify-content: center;
+  padding-bottom: 0.5rem;
+}
+
+.information-container__item{
+  margin: 0;
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+ 
+}
+
+.bordered{
+  border-right: rgba(112, 112, 112, 0.42) solid thin;
+  border-left: rgba(112, 112, 112, 0.42) solid thin;
```

Comment:
use variable for colors, use correct shorthand, use px instead of named border widths